src/migrations/20230116162612-add-sats-amounts-to-receipts.ts

```diff
@@ -0,0 +1,91 @@
+module.exports = {
+  async up(db) {
+    const txTypes = ["invoice", "onchain_receipt"]
+
+    const collection = db.collection("medici_transactions")
+
+    try {
+      const allTxns = await collection.aggregate([
+        {
+          $match: {
+            type: { $in: txTypes },
+            satsAmount: { $exists: false },
+            currency: "BTC",
+          },
+        },
+        {
+          $group: {
+            _id: "$_journal",
+            // 'debit' here would be the full amount received by lnd
+            debit: { $max: "$debit" },
```

Comment:
if we want to update user records.. shouldn't be this credit? (debit could be split between liabilities and bank account)